crates/edr_provider/src/data.rs

```diff
@@ -2493,15 +2529,19 @@ mod tests {
 
         assert_eq!(fixture.console_log_calls().len(), 0);
 
+        let signed_transaction = fixture
+            .provider_data
+            .sign_transaction_request(transaction)?;
+
         fixture.provider_data.set_auto_mining(false);
-        fixture.provider_data.send_transaction(transaction)?;
+        fixture.provider_data.send_transaction(signed_transaction)?;
         let (block_timestamp, _) = fixture.provider_data.next_block_timestamp(None)?;
         let prevrandao = fixture.provider_data.prev_randao_generator.next_value();
-        fixture
+        let result = fixture
             .provider_data
             .mine_block(block_timestamp, Some(prevrandao))?;
 
-        let console_log_calls = fixture.console_log_calls();
+        let console_log_calls = result.console_log_inputs;
```

Comment:
Nit: if you make changes anyway, could you rename `console_log_calls` to `console_log_inputs`? Otherwise feel free to ignore

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/4781/commits/498f131e4d40a3a8b697bdcfc68c6c8ea6e700db